src/pages/gallery/index.astro

```diff
@@ -4,12 +4,17 @@ import "./gallery.css";
 import { getSession } from "../../lib/game-saving/account";
 import Navbar from "../../components/navbar-main";
 import StandardHead from "../../components/standard-head.astro";
-import { getGalleryGames } from "../../lib/game-saving/gallery";
+import { GameMetadata, getGalleryGames } from "../../lib/game-saving/gallery";
 import Gallery from "./gallery";
 
 const session = await getSession(Astro.cookies);
 
-const games = getGalleryGames();
+const _games = getGalleryGames().map(game => ({ ...game, title: game.title.trim(), smallTitle: game.title.toLowerCase(), smallAuthor: game.author.toLowerCase() }));
+const games = _games.sort((a, b) => Date.parse(b.addedOn) - Date.parse(a.addedOn))
```

Comment:
Sorry - I don't understand why we have a separate/different sorting heuristic here (IMO we should always use the same gallery/game sorting code, period)